programs/programs/co/medicaid/family_planning_services/tests/test_family_planning_services.py

```
@@ -0,0 +1,331 @@
+    return FamilyPlanningServices(
+        mock_screen,
+        mock_program,
+        make_data(medicaid_eligible),
+        Dependencies() if missing_dependencies is None else missing_dependencies,
+    )
+
+
+def eligibility_with_members(count):
+    """An `Eligibility` shaped the way `eligible()` hands it to `household_eligible`."""
+    e = Eligibility()
+    for i in range(count):
```

Ruff (0.16.1)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)